//申请加入公会
message DseGuildApplyJoin
{
	optional int32	res = 1;
}
